cartridges/app_storefront_base/cartridge/controllers/Hello.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @namespace Hello;
+ */
+
+var server = require('server');
+/**
+ * @name Base/Hello-World;
+ */
+
+server.get("World", function (req, res, next) {
```

Comment:
You can delete this controller, since we no longer need to make changes to app_storefront_base